### PR TITLE
VZ-9623, VZ-9925. Documentation update for Cert-Manager changes for 1.6

### PR DESCRIPTION
**Pull request by michael-cico**
Update `Customizing Certificates` section to reference new APIs for Cert-Manager configuration, and add a new section and examples for using a customer-managed Cert-Manager with Verrazzano.

### CODE REVIEW

File: content/en/docs/networking/security/certificates.md [hunk 1]
```diff
@@ -188,3 +189,103 @@ rate limit exceptions on certificate generation.
...
+{{< clipboard >}}
+<div class="highlight">
+
+```
+apiVersion: install.verrazzano.io/v1beta1
+kind: Verrazzano
+metadata:
+  name: custom-cm-example
+spec:
+  profile: dev
```
Comment (mbenmclean) [inline content/en/docs/networking/security/certificates.md:207]: Change to prod. We want to be encouraging people to use the prod profile over dev

File: content/en/docs/networking/security/certificates.md [hunk 2]
```diff
@@ -188,3 +189,103 @@ rate limit exceptions on certificate generation.
...
+{{< clipboard >}}
+<div class="highlight">
+
+```
+apiVersion: install.verrazzano.io/v1beta1
+kind: Verrazzano
+metadata:
+  name: custom-cmres-example
+spec:
+  profile: dev
```
Comment (mbenmclean) [inline content/en/docs/networking/security/certificates.md:234]: same comment about prod profile

File: content/en/docs/networking/security/certificates.md [hunk 3]
```diff
@@ -188,3 +189,103 @@ rate limit exceptions on certificate generation.
...
+{{< clipboard >}}
+<div class="highlight">
+
+```
+apiVersion: install.verrazzano.io/v1beta1
+kind: Verrazzano
+metadata:
+  name: custom-cm-letsencrypt-example
+spec:
+  profile: dev
```
Comment (mbenmclean) [inline content/en/docs/networking/security/certificates.md:267]: same comment

File: content/en/docs/networking/security/certificates.md [hunk 4]
```diff
@@ -188,3 +189,103 @@ rate limit exceptions on certificate generation.
...
+      letsEncrypt:
+        emailAddress: jane.doe@mycompany.com
+        environment: staging
+    dns:
+      oci:
+        ociConfigSecret: oci
+        dnsZoneCompartmentOCID: ocid1.compartment.oc1.....
+        dnsZoneOCID: ocid1.dns-zone.oc1.....
+        dnsZoneName: example.com
+    certManagerWebhookOCI:
```
Comment (mbenmclean) [inline content/en/docs/networking/security/certificates.md:282]: Is this necessary for it to work?

Comment (michael-cico) [reply content/en/docs/networking/security/certificates.md:282]: yes.  We need them to tell the webhook where the CM and clusterResourceNamespace are for RBAC configuration.

File: content/en/docs/networking/security/certificates.md (hunk 1 above)
Comment (michael-cico) [reply content/en/docs/networking/security/certificates.md:207]: It was dev before in the other examples as well; I think the idea is to let them copy and paste simple examples.  `prod` is a bit more heavyweight.

File: content/en/docs/networking/security/certificates.md (hunk 2 above)
Comment (michael-cico) [reply content/en/docs/networking/security/certificates.md:234]: same comment

File: content/en/docs/networking/security/certificates.md (hunk 4 above)
Comment (mbenmclean) [reply content/en/docs/networking/security/certificates.md:282]: Oh ok I didn't know that I'll have to update the a-la-carte pipeline to reflect

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**